sources/core/Stride.Core.Design/Editor/RiderPathLocator.cs

```diff
@@ -0,0 +1,365 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Win32;
+using Newtonsoft.Json;
+using Directory = System.IO.Directory;
+using Environment = System.Environment;
+using File = System.IO.File;
+using Path = System.IO.Path;
+
+// ReSharper disable UnassignedField.Local
+// ReSharper disable InconsistentNaming
+// ReSharper disable UnassignedField.Global
+// ReSharper disable MemberHidesStaticFromOuterClass
+
+namespace Stride.Core.Editor
+{
+    /// <summary>
+    /// This code is a modified version of the JetBrains resharper-unity plugin listed under Apache License 2.0 license:
```

Comment:
Stride is under MIT, I'm not too up to date on that stuff but can you really just include stuff from Apache like that and call it a day ?

Reply:
Both Apache and MIT are permissive. And I am the author of original code. I am going to ask our legal department, unfortunately it might take week or two.

Reply:
@van800 if you are the author of the code, then you can with it whatever you want. the license applies to people who don't own the copyright.

Reply:
I have got a response from the legal team:
```
Hi, if you signed the CLA and the code was then sent to them without it being labelled as "not a contribution", then it constitutes a contribution and is fully governed by the terms of the CLA, not Apache license. So yes you are allowed to do it, and its now governed by the rules of the CLA. 
```

Reply:
I would probably remove mention of Apache License in that comment.